defaults/test_default.yaml

```diff
@@ -216,6 +216,7 @@ stress_image:
   kcl: 'scylladb/hydra-loaders:kcl-jdk8-20210526-ShardSyncStrategyType-PERIODIC'
   harry: 'scylladb/hydra-loaders:cassandra-harry-jdk11-20220816'
   latte: 'scylladb/hydra-loaders:latte-rust1_73-20231025'
+  cql-stress-cassandra-stress: 'scylladb/hydra-loaders:cql-stress-cassandra-stress-20231130'
```

Comment:
It tool itself has some release cadence and versions we can use them and not dates

Also it would be nice if the docker images would be built on the tool repo

Gemini now does it.